mystery-deckbuilder/Assets/Scripts/NPC/NPC.cs

```diff
@@ -0,0 +1,57 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class NPC : MonoBehaviour
+{
+    //Variables 
+    public string name;
+    [Range(0.0f, 1.0f)]public float affinity_1;
+    [Range(0.0f, 1.0f)]public float affinity_2;
+    [Range(0.0f, 1.0f)]public float affintiy_3;
+    [Range(0.0f, 1.0f)]public float startingPatience;
+    [Range(0.0f, 1.0f)]public float startingCompliance;
+    [Range(0.0f, 1.0f)]public float currentPatience;
+    [Range(0.0f, 1.0f)]public float currentCompliance;
+    public DialogueTree diaglogue;
+
+    //Get current patience
+    public float getPatience()
+    {
+        return currentPatience;
+    }
+
+    //Set current patience
+    public void setPatience(float x)
+    {
+        currentPatience = x;
+    }
+
+    //Get current compliance
+    public float getCompliance()
+    {
+        return currentCompliance;
+    }
+
+    //Set current compliance
+    public void setCompliance(float x)
+    {
+        currentCompliance = x;
+    }
```

Comment:
As per style guide, we are trying to use PascalCase instead of camelCase for methods. i.e. SetPatience and GetCompliance